### DISCUSSION TITLE
[Valheim] Bug mod install

### DISCUSSION THREAD

**Discussion by sosnax1** [GameServerManagers/LinuxGSM#3604] [General]
After installing mods, using command ./vhserver mi  it doesn't start automaticly...

`Please open start_server_bepinex.sh in a text editor and provide the correct exe`

**Comment by johnoclockdk** [comment]
that easy change start_server.sh to start_server.sh.bak and change start_game_bepinex.sh to start_server.sh

**Reply by sosnax1** [reply]
that did the trick, for starting the server, but now mods bepinex is not started...

**Reply by johnoclockdk** [reply]
have you updated the mod after homes update

**Reply by sosnax1** [reply]
yes, freshly downoladed today, but even without mods, i cant just see bepinex loaded, and as i remember on windows machine there were bepinex meseages acknowledig it being loaded. Swapping start_server.sh does start server, but does not load bepinex (as excepted)

**Reply by sosnax1** [reply]
I have managed to run it XD. With BepInEx, just manually downoladed, and swapped files... But now i can't connect, realised LinuxGSM have old version (5.4.15.0) of bepinex, updated that manually. Still can't connect. This game server is real pain in the ass to setup...

**Reply by sosnax1** [reply]
Yep, and by changing start files linuxGSM is unable to change ports etc. So if anyone ll have same problem, after swapping bepinex files with latest release u have to edit start_server_bepinex.sh and put server details like you wuld put it in normall linuxGSM instance file.
